app/Models/Activity.php

```diff
@@ -80,8 +81,11 @@ public static function boot()
     {
         parent::boot();
 
-        static::addGlobalScope('approve', function (Builder $builder) {
```

Comment:
Why "addGlobalScope" is removed from "Activity" model while it still exists in "IndependentActivity".